model/src/main/java/hu/blackbelt/judo/meta/jsl/validation/JslDslValidator.xtend

```diff
@@ -81,6 +88,13 @@ class JslDslValidator extends AbstractJslDslValidator {
 	public static val DEFAULT_TYPE_MISMATCH = ISSUE_CODE_PREFIX + "DefaultValueTypeMismatch"
```

Comment:
No all checks are covered with tests. Please add missing test case, every check have to be tested.

Reply:
Done